interpreter/interpreter.py

```diff
@@ -43,6 +43,9 @@
 from rich.markdown import Markdown
 from rich.rule import Rule
 
+# create an instance of the speech assistant
+assistant = SpeechAssistant(wakeup_word="interpreter")
```

Comment:
would this one still create a listener to the Mic if there is not enable voice?

Reply:
yeah sorry it was there abilteraly i will put it somewhere more usefull no jokes 😁